mw-webapp/src/component/accordion/AccordionContent/AccordionContent.tsx

```diff
@@ -18,7 +23,10 @@ export interface AccordionContentProps {
  */
 export const AccordionContent = (props: AccordionContentProps) => {
   return (
-    <RadixAccordionContent className={styles.accordionContent}>
+    <RadixAccordionContent
+      className={styles.accordionContent}
+      data-cy={props.dataCy ?? "accordion-content"}
```

Comment:
Great idea, but let's skip default data-cy attributes for now, at least until we will start to write e2e tests

Reply:
In the Button component the same situation. And as I remember without default value tests don't work. But maybe I'm wrong...

Reply:
Yes, data-cy is for tests, I need unique data.

Reply:
So could we get rid of default data-cy attributes in components?

Reply:
Done.